### PR TITLE
Add services to VM Details

### PR DESCRIPTION
**Pull request by rawagner**

### CODE REVIEW

**Comment by rawagner** [comment]
services

File: src/components/Details/VmDetails/tests/VmDetails.test.js [hunk 1]
```diff
@@ -62,6 +62,11 @@ describe('<VmDetails />', () => {
     expect(component).toMatchSnapshot();
   });
 
+  it('renders correctly with services', () => {
+    const component = render(testVmDetails(vmFixtures.downVm, { services }));
```
Comment (suomiy) [inline src/components/Details/VmDetails/tests/VmDetails.test.js]: it only show the loading. Can you please put some rendered services?

File: src/components/Details/VmDetails/VmDetails.js [hunk 2]
```diff
@@ -312,6 +316,22 @@ export class VmDetails extends React.Component {
                     template={this.state.template}
                   />
                 </dd>
+                <dt>Services</dt>
+                <dd>
+                  {services ? (
+                    services
+                      .filter(service => get(service, ['spec', 'selector', TEMPLATE_VM_NAME_LABEL]) === getName(vm))
```
Comment (suomiy) [inline src/components/Details/VmDetails/VmDetails.js]: this should go probably to `selectors/service/`

File: src/components/Details/VmDetails/tests/VmDetails.test.js (hunk 1 above)
Comment (rawagner) [reply src/components/Details/VmDetails/tests/VmDetails.test.js]: fixed, fixture is now rendering services

File: src/components/Details/VmDetails/VmDetails.js (hunk 2 above)
Comment (rawagner) [reply src/components/Details/VmDetails/VmDetails.js]: moved to `selectors`

**Comment by rawagner** [comment]
@suomiy I have also created new component `Services`

**Comment by suomiy** [comment]
> I have also created new component `Services`

+1

**Pull request merged**